docarray/array/array/io.py

```diff
@@ -311,21 +310,22 @@ def to_base64(
     @classmethod
     def from_json(
         cls: Type[T],
-        file: Union[str, bytes, bytearray],
+        file: str,
     ) -> T:
         """Deserialize JSON strings or bytes into a DocArray.
 
         :param file: JSON object from where to deserialize a DocArray
         :return: the deserialized DocArray
         """
         json_docs = json.loads(file)
-        return cls([cls.document_type.parse_raw(v) for v in json_docs])
+        return cls([cls.document_type(**v) for v in json_docs])
```

Comment:
`parse_raw` was expecting a string but now it receives a dict.

This is because the way we were doing json encoding -> '["{text: smth}"]' so `parse_raw` was taking "{text: smth}"

Now it would be '[{text: smth}]' so we pass a dict instead of a string.

Reply:
okay great

Reply:
yes my bad you are right

Reply:
but should not we use orjson load rather than json.load ? Otherwise tensor loading will be problematic or slow